Doc/library/inspect.rst

```diff
@@ -1165,7 +1165,7 @@ The interpreter stack
 
 When the following functions return "frame records," each record is a
 :term:`named tuple`
-``FrameInfo(frame, filename, lineno, function, code_context, index)``.
+``FrameInfo(frame, filename, lineno, end_lineno, column, end_column, function, code_context, index)``.
```

Comment:
hm, this would break users who unpack the tuple. Can we do that?

Reply:
Unclear, more closely to no. Check my comment on the issue.

Reply:
What's the issue? #88166 looks unrelated.